flax/nnx/summary.py

```diff
@@ -130,14 +165,14 @@ def __str__(self):
 
 @dataclasses.dataclass
 class CallInfo:
+  call_order: int
   object_id: int
   type: type
   path: statelib.PathParts
-  input_args: tuple[tp.Any, ...]
-  input_kwargs: dict[str, tp.Any]
+  inputs_repr: str
```

Comment:
We can reuse the table representation as a hashable value to filter for when we're adding duplicate rows to the table.